hr_timesheet_sheet_attendance/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': 'HR Timesheet Sheet Attendance',
+    'version': '11.0.0.0.0',
+    'category': 'Human Resources',
+    'sequence': 80,
+    'license': 'AGPL-3',
+    "author": "BizzAppDev, "
```

Comment:
@bizzappdev Add `website` key, URL like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)